### ISSUE TITLE
Duplicate `@wry/trie` in dependency tree

### ISSUE THREAD

**Issue by levrik** [apollographql/apollo-client#11041]
The `@wry/trie` dependency is currently duplicated in Apollo's dependency tree.
An update to `optimism@0.17.5` would be required to get rid of this duplication, see https://github.com/benjamn/optimism/pull/551

**Comment by alessbell** [comment]
Good catch, @levrik! This will be fixed in our 3.8 release candidate which is imminent 🎉 , so I'll close this out. Thanks!
